components/clearbutton/stories/clearbutton.stories.js

```diff
@@ -1,13 +1,13 @@
 import { Sizes } from "@spectrum-css/preview/decorators";
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
-import { isDisabled, isQuiet, size, staticColor } from "@spectrum-css/preview/types";
+import { isDisabled, size } from "@spectrum-css/preview/types";
```

Comment:
If we use the `isHovered` and `isActive` types, we could have those controls in the table for Storybook 🤔 

<img width="1174" height="179" alt="Screenshot 2025-07-23 at 4 38 50 PM" src="https://github.com/user-attachments/assets/32a3bd36-2116-4eac-81bf-c81a65faa98b" />

There'd be a couple of things left to do, like add those args to the default list so the boolean is set in the control table, change around a few things in the template and test files, but I think it'd be nice to get those controls in.